testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/javascript/JavascriptAdapterTest.java

```diff
@@ -386,21 +389,23 @@ public void implicitFlowRefreshTokenTest() {
 
     @Test
     public void implicitFlowOnTokenExpireTest() {
-        RealmRepresentation realm = adminClient.realms().realm(REALM_NAME).toRepresentation();
-        Integer storeAccesTokenLifespan = realm.getAccessTokenLifespanForImplicitFlow();
-        try {
-            realm.setAccessTokenLifespanForImplicitFlow(5);
-            adminClient.realms().realm(REALM_NAME).update(realm);
-
-            setImplicitFlowForClient();
-            testExecutor.logInAndInit(defaultArguments().implicitFlow(), testUser, this::assertInitAuth)
-                  .addTimeSkew(-5); // Move in time instead of wait
-
-            waitUntilElement(eventsArea).text().contains("Access token expired");
-        } finally {
-            // Get to origin state
-            realm.setAccessTokenLifespanForImplicitFlow(storeAccesTokenLifespan);
-            adminClient.realms().realm(REALM_NAME).update(realm);
+        try (RealmAttributeUpdater rau = new RealmAttributeUpdater(adminClient.realms().realm(REALM_NAME))
+                .setAccessTokenLifespanForImplicitFlow(3)
```

Comment:
Should we consider setting that even lower? Since we're only testing expiration in this test, maybe even 1 second?